tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -143,8 +152,17 @@
       set -o pipefail
 
       cd $GOPATH/src/k8s.io/cloud-provider-openstack
-      mkdir /var/log/csi-pod
-      go test -v ./cmd/tests/cinder-csi-e2e-suite/cinder_csi_e2e_suite_test.go -ginkgo.v -ginkgo.progress -ginkgo.skip="\[Disruptive\]" -ginkgo.focus="\[cinder-csi-e2e\]" -ginkgo.noColor -timeout=0 -report-dir="/var/log/csi-pod" | tee "/var/log/csi-pod/cinder-csi-e2e.log"
+      mkdir -p /var/log/csi-pod
+      # TODO(chrigl): No idea why both tests fail in CI. On a real OpenStack both pass.
+      /tmp/kubernetes/test/bin/e2e.test \
+        -storage.testdriver=tests/e2e/csi/cinder/test-driver.yaml \
+        -ginkgo.focus='External.Storage' \
+        -ginkgo.skip='\[Disruptive\]|\[Testpattern:\s+Dynamic\s+PV\s+\(default\s+fs\)\]\s+provisioning\s+should\s+mount\s+multiple\s+PV\s+pointing\s+to\s+the\s+same\s+storage\s+on\s+the\s+same\s+node|\[Testpattern:\s+Dynamic\s+PV\s+\(default\s+fs\)\]\s+provisioning\s+should\s+provision\s+storage\s+with\s+any\s+volume\s+data\s+source\s+\[Serial\]' \
```

Comment:
do we have a general place to contain such workaround?

Reply:
No that I'm aware of. Should we track it as an issue?

Reply:
yes, definitely

Reply:
Created a tracking issue: #1871